modules/rest-client/pom.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.apache.ignite</groupId>
+    <artifactId>ignite-parent</artifactId>
+    <version>1</version>
+    <relativePath>../../parent/pom.xml</relativePath>
+  </parent>
+
+  <artifactId>ignite-rest-client</artifactId>
+  <version>3.0.0-SNAPSHOT</version>
+
+  <dependencies>
+
+    <!--client specific dependencies, versions cannot be moved to the parent pom-->
```

Comment:
Is this module (and this file) auto-generated?

Reply:
Actually, this is a valuable note. 

This module (just module, not code) and pom are not generated. The plugin section in this pom configures the source code generation. This is done for integration tests for the generated client in the runner module.  

From one side, the client should be generated where it is really needed. In our case, it is inside the cli module. But the Open API spec is generated by the rest module and I think that there should be an integration test for the client that is generated from this Open API spec. And these checks have to be done by the server.

I can add a comment to the pom or drop this module and create a ticket for integration tests that will generate the client somewhere inside and not include it as a module.

Reply:
> drop this module and create a ticket for integration tests that will generate the client somewhere inside and not include it as a module.

That would be very much preferable, thank you